File: palindrome.js
let promptSync = require('prompt-sync')();

function palindrome(word) {
    for (let i = word.length - 1; i >= Math.floor(word.length / 2); i--) {
        if (word[i] == word[word.length - i - 1]) continue;
        else return false;
    }
    return true;
}

function Main() {
    try {
        console.log('See if the word is a palindrome');
        process.stdout.write('Insert a word: ');
        let r = promptSync();
        let sender = palindrome(r);
        console.log(sender);
    } catch (err) {
        console.log('An error ocurred during the process...')
        console.log(err.message);
    }
}
Main();

File: Python/palindrome.py
import math

def palindrome():
    word = input("Insert a word: ")
    i = len(word)
    while i >= math.floor((len(word)) / 2):
        i -= 1
        if word[i] == word[len(word)-i-1]:
            continue
        else:
            return False

    return True


print(palindrome())